sim/pokemon.ts

```diff
@@ -560,8 +560,10 @@ export class Pokemon {
 		return this.foes().filter(foe => this.battle.isAdjacent(this, foe));
 	}
 
-	getMoveTargets(move: Move, target: Pokemon): Pokemon[] {
+	getMoveTargets(move: Move, target: Pokemon): {hit: Pokemon[], pp: Pokemon[]} {
```

Comment:
I feel like `pressureTargets` is a better name than `pp`.

`targets`, `pressureTargets`.